lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -751,36 +847,94 @@ bool ProxyRenderDelegate::getInstancedSelectionPath(
         indexPath = _sceneDelegate->GetPathForInstanceIndex(rprimId, 0, nullptr);
         usdPath = _sceneDelegate->ConvertIndexPathToCachePath(indexPath);
     }
+
+    // The "Instances" point instances pick mode is not supported for
+    // USD_IMAGING_API_VERSION < 13 (core USD versions earlier than 20.05), so
+    // no setting of topLevelPath or topLevelInstanceIndex here.
```

Comment:
What behavior is expected in this scenario?

Reply:
You definitely will not be able to pick individual point instances for USD 20.02. If you're in `PointInstancer` or `Instances` pick mode, we'll still be able to identify if you've picked an instance proxy and can traverse up to the nearest non-proxy prim. In `Prototypes` pick mode, the behavior should be no different from the current behavior. We'll also still apply selection kind filtering afterwards in all cases.

This was less about trying to support a specific behavior for USD 20.02 and just trying to make sure the functionality degrades somewhat gracefully.